mesonbuild/environment.py

```diff
@@ -124,6 +124,24 @@
 
 CompilersDict = T.Dict[str, Compiler]
 
+COMPILER_OPTS_TO_ENV = {
+    'c': 'CFLAGS',
+    'cpp': 'CXXFLAGS',
+    'cuda': 'CUFLAGS',
+    'objc': 'OBJCFLAGS',
+    'objcpp': 'OBJCXXFLAGS',
+    'fortran': 'FFLAGS',
+    'd': 'DFLAGS',
+    'vala': 'VALAFLAGS',
+    'rust': 'RUSTFLAGS',
+    'pkg_config_path': 'PKG_CONFIG_PATH',
```

Comment:
I don't think 'pkg_config_path' should be here.

Reply:
You're right, I keep rebasing that back *in* somehow :(